tests/main/network-retry/task.yaml

```diff
@@ -29,8 +29,8 @@ execute: |
     fi
 
     echo "Try to install a snap with broken DNS"
-    if snap install test-snapd-tools; then
```

Comment:
I think the purpose of this test is to hit the store for real.

If you want to get better efficiency perhaps we should have a snap like `test-snapd-sh-online` that is only in the store?

Reply:
In this case it is not needed to use test-snapd-sh due to it is not being installed.

@zyga, just to clarify, the test-snapd-sh which is in the store is not pluging any interface, it is a reduced one like it is defined in https://code.launchpad.net/~snappy-dev/snappy-hub/test-snapd-sh and like it is defined here as well #7815

The idea is that once we merge the #7815 and this one the performance of the tests should be much better. 

I'll add thiat description to the PR.

Reply:
@zyga please confirm if that explanation makes sense and it is enough. Thanks for the review.